src/routes/reporting.ts

```diff
@@ -0,0 +1,15 @@
+import * as express from 'express';
+import rawReportingExportController from 'src/controllers/rawReportingExportController';
+import {authorize} from "../middlewares/authMiddleware";
+import {UserType} from "../models/userModel";
+
+const reportingExportRoutes =  (reportingController: rawReportingExportController) => {
+    const router = express.Router();
+    router.route('/purchase-events').get(

```

Comment:
Here since we are sending a body in the request, could we please change the request method from "GET" to "POST".

So just changing the word get to post as follows in the highlighted line:
`router.route('/purchase-events').post(`

@yagi527 
@fszwedo fyi